### PR TITLE
Teams SPFx apps in Outlook/M365 app preview

### PR DESCRIPTION
**Pull request by erikadoyle**
Update M365 apps content with details on [SPFx 1.16 support](https://devblogs.microsoft.com/microsoft365dev/announcing-general-availability-of-sharepoint-framework-1-16-enabling-spfx-across-microsoft-365-platform//)

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/eeffa57a26a7ca1494d35b0c97b0bc89608c3aeb/msteams-platform%2Fm365-apps%2Fextend-m365-teams-personal-tab.md) | 95 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/303826b8-08a4-440b-b34f-ee0a110c2029) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/eeffa57a26a7ca1494d35b0c97b0bc89608c3aeb/msteams-platform%2Fm365-apps%2Foverview.md) | 92 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/16a7aaa5-4366-47f8-a767-7eff7e6b8162) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by erikadoyle** [comment]
Hi @v-habibaumme when you have a chance, could you help to review/merge? This feature was [recently ](https://devblogs.microsoft.com/microsoft365dev/updated-preview-of-the-sharepoint-framework-1-16/) announced, so it'd be great to have the docs reflect that. Thanks!

**Comment by erikadoyle** [comment]
@surbhigupta12 @v-habibaumme could someone help review/merge this? It's a small but important update for M365 docs. Thanks!

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md [hunk 1]
```diff
@@ -62,6 +62,15 @@ To start with a [personal tab](https://github.com/OfficeDev/TeamsFx-Samples/tree
...
 From here, you can skip ahead to [sideload your app in Teams](#sideload-your-app-in-teams) and preview your app in Outlook and Office. (The app manifest and TeamsJS API calls have already been updated for Microsoft 365.)
 
+### SharePoint Framework (SPFx) apps
+
+Starting with version 1.16 (preview) of [SharePoint Framework](/sharepoint/dev/spfx/integrate-with-teams-introduction) (SPFx), Teams personal tabs built and hosted with SPFx are also supported in Outlook and Office. To update a SPFx Teams personal tab app, you'll need to first [install the latest preview version of SPFx](/sharepoint/dev/spfx/release-1.16#install-the-latest-preview-version), and then follow the steps in this article to:
+
+1. [Update the app manifest](#update-the-app-manifest)
+1. [Update SDK references](#update-sdk-references)
+
+From there, you can skip ahead to [Sideload your app in Teams](#sideload-your-app-in-teams) to preview your SPFx personal tab app running in Outlook and Office. (You can skip the sections on configuring Content Security Policy and Azure AD app registration.)
```
Comment (v-ypalikila) [inline msteams-platform/m365-apps/extend-m365-teams-personal-tab.md:72]:
```suggestion
Starting with version 1.16 (preview) of [SharePoint Framework](/sharepoint/dev/spfx/integrate-with-teams-introduction) (SPFx), Teams personal tabs built and hosted with SPFx are also supported in Outlook and Office. To update a SPFx Teams personal tab app, follow these steps:

1. [Install the latest preview version of SPFx](/sharepoint/dev/spfx/release-1.16#install-the-latest-preview-version)
1. [Update the app manifest](#update-the-app-manifest)
1. [Update SDK references](#update-sdk-references)

After updating the SDK references, [Sideload your app in Teams](#sideload-your-app-in-teams) to preview your SPFx personal tab app running in Outlook and Office.```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cbd60bf3163736008de71ffd33b354cbef593f80/msteams-platform%2Fm365-apps%2Fextend-m365-teams-personal-tab.md) | 95 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3b05233f-2fda-49bd-8b32-324dc260a633) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cbd60bf3163736008de71ffd33b354cbef593f80/msteams-platform%2Fm365-apps%2Foverview.md) | 92 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1a675731-4116-4b41-849b-1cd6cc1137d6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md (hunk 1 above)
Comment (erikadoyle) [reply msteams-platform/m365-apps/extend-m365-teams-personal-tab.md]: Good suggestion--I manually incorporated it, as the suggested change has three stray tick marks (```) at the end

**Comment by erikadoyle** [comment]
Thanks @v-ypalikila, I've incorporated your suggestion

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d0bdb2c0edc2943712516c7d0b64d35ba71ba1fe/msteams-platform%2Fm365-apps%2Fextend-m365-teams-personal-tab.md) | 95 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a4be2199-683f-4009-be16-4103493cc932) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d0bdb2c0edc2943712516c7d0b64d35ba71ba1fe/msteams-platform%2Fm365-apps%2Foverview.md) | 92 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e0880589-a6a5-4f8f-8639-0b242948911b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/739f36552d0f469cedb819cd4ace7019e23d3b9a/msteams-platform%2Fm365-apps%2Fextend-m365-teams-personal-tab.md) | 95 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dffd814f-73ff-4f11-a46b-f9a543529fa7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/739f36552d0f469cedb819cd4ace7019e23d3b9a/msteams-platform%2Fm365-apps%2Foverview.md) | 92 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b9852b55-cdf8-4fa4-a313-1c1f6f2e6c1f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-ypalikila — DISMISSED** [review]
Approved.

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/739f36552d0f469cedb819cd4ace7019e23d3b9a/msteams-platform%2Fm365-apps%2Fextend-m365-teams-personal-tab.md) | 95 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/249887bd-64ea-493f-9f86-599d97727b30) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/739f36552d0f469cedb819cd4ace7019e23d3b9a/msteams-platform%2Fm365-apps%2Foverview.md) | 92 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/93c5743a-5d15-4bda-ba6b-3c28aced1ca6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/49c3752d3553aab3c59d7dfc7d0a5853af1c7a62/msteams-platform%2Fm365-apps%2Fextend-m365-teams-personal-tab.md) | 95 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9c92c37b-d9a8-40d8-ba01-e9e667746eed) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/49c3752d3553aab3c59d7dfc7d0a5853af1c7a62/msteams-platform%2Fm365-apps%2Foverview.md) | 92 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2b0a8503-bdc7-4690-8ee9-80828c4342a4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**